charts/common/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: library
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.7
+version: 0.1.8
```

Comment:
Hello,
My recent changes going to change the version to version: 0.1.8
Would you please update your version to 0.1.9
P\S Most probably my PR will be merged by EOD.
Thanks in advance!